### DISCUSSION TITLE
Circular dependencies

### DISCUSSION THREAD

**Discussion by filipw01** [crxjs/chrome-extension-tools#339] [General]
Since 4.0.1-20 I started having issues with circular dependencies which break vite development mode? Is this something not supported by the plugin or is this a bug that should be fixed? @jacksteamdev

**Comment by filipw01** [comment]
Sorry that's not the issue that is breaking, these are just warnings

**Comment by jacksteamdev** [comment]
@filipw01 Based on the error in #338, this looks like a Rollup limitation. We use Rollup to adapt Vite's network-based HMR to the file system. I'm open to ideas!
